pkg/activator/handler/handler.go

```diff
@@ -158,11 +158,18 @@ func (a *activationHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		Host:   host,
 	}
 
+	_, epWaitSpan := trace.StartSpan(r.Context(), "endpoint_wait")
+	epWaitStart := time.Now()
+	epWaitEnd := func() {
+		epWaitSpan.End()
+		a.logger.Infof("Waiting for endpoint took %v time", time.Since(epWaitStart))
```

Comment:
Shouldn't we only log this in the successful case? In the error case we don't know if we even waited for endpoints to appear so this could be confusing in the logs.

Reply:
Also: This isn't really only waiting for endpoints but also waiting for capacity to be available in general. Should this rather be `capacity_wait`?

On a broader scheme: Should we propagate context into `Try` to be able to distinguish better?

Reply:
I agree with the logs + rename. Right now we could propogate ctx in to try but I am hoping to decouple probing from individual requests so that'll end up getting removed anyways.